client/components/account-status/style.scss

```diff
@@ -0,0 +1,62 @@
+/** @format */
+
+.woocommerce-account-status-header {
+	display: flex;
+	flex-direction: row;
+}
+.woocommerce-account-status-header div.title {
+	font-size: 16px;
+}
+.woocommerce-account-status-header div {
+	padding-right: 16px;
+}
+.woocommerce-account-status-header-controls {
+	flex-grow: 3;
+	text-align: right;
+}
+.woocommerce-account-status-item {
+	align-items: center;
+	display: flex;
+	flex-direction: row;
+	font-style: normal;
+	font-weight: normal;
+	height: 2em;
+	padding-right: 16px;
+}
+.woocommerce-account-status-item-label {
+	color: $gray-900;
+	flex-grow: 0;
+	font-size: 13px;
+	line-height: 16px;
+	margin: 0 4px;
+	width: 130px;
```

Comment:
We probably need media queries and custom styling for mobile version to make it more responsive.

Reply:
I actually decided to remove them completely and use a `Flex` component recently became available.